configuration/views.py

```diff
@@ -121,6 +122,7 @@ def uncon_studies_form(self, request, **kwargs):
             available_suppression_method_types=available_suppression_method_types,
             available_suppression_method_sub_types=available_suppression_method_sub_types,
             available_processing_main_domain_types=available_processing_main_domain_types,
+            available_specific_paradigm_type=available_specific_paradigm_type,
```

Comment:
you also need to add it in the serializer processing this view or it won't happen